ui/packages/consul-ui/app/components/auth-form/index.hbs

```diff
@@ -154,7 +154,7 @@ as |TabState IgnoredGuard IgnoredAction tabDispatch tabState|>
       @nspace={{or this.value.Namespace @nspace}}
       @partition={{or this.value.Partition @partition}}
       @type={{if this.value.Name 'oidc' 'secret'}}
-      @value={{if this.value.Name this.value.Name this.value}}
```

Comment:
If its an OIDC flow then pass the entire ODIC object through instead of just the name (the object contains the partition name received from the backend, so we are sure thats correct)